Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```diff
@@ -0,0 +1,1041 @@
+                      'OwnerID', 'Priority', 'Origin', 'Status', 'Reason', 'IsEscalated', 'SuppliedPhone',
+                      'SuppliedCompany', 'SuppliedEmail', 'ContactEmail', 'ContactId', 'AccountId']
+        cases = client.queryObjects(properties, 'Case', condition).get('records')
+        return objectToEntry(client, cases)
+
+    elif args.get('oid'):
+        cases = client.getObject(f"Case/{args.get('oid')}")
+        return objectToEntry(client, cases)
+
+    else:
+        return_error('You must specify object ID or a Case Number')
+
+
+def get_user_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    response = client.getUser(args.get('oid', None), args.get('caseNumber', None))
+    return objectToEntry(client, response)
+
+
+def get_case_comment_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    result = client.getCaseComment(args.get('oid', None), args.get('caseNumber', None))
+
+    return objectToEntry(client, result.get('records', []))
+
+
+def get_org_name_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('caseNumber', None):
+        condition = f"CaseNumber='{args.get('caseNumber', None)}'"
+        properties = ['ID', 'CaseNumber', 'AccountId']
+        cases = client.queryObjects(properties, 'Case', condition).get("records")
+        conditionA = f"Id='{cases[0].get('AccountId')}'"
+        propertiesA = ['Id', 'Name']
+        usersA = client.queryObjects(propertiesA, "Account", conditionA).get("records")
+    else:
+        return_error('You must specify a Case Number')
+    return objectToEntry(client, usersA)
+
+
+def post_case_comment_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    results = client.postCaseComment(args.get('public'), args.get(
+        'oid', None), args.get('caseNumber', None), args.get('text', None))
+    if results.get('success', None):
+        caseComment = client.getObject(f"{results.get('id')}")
+    else:
+        return_results(f"Unable to post case comment. Error Encountered was:{json.dumps(results.get('errors', None))}")
+
+    return objectToEntry(client, caseComment)
+
+
+def create_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    data = {
+        "Subject": args.get('subject'),
+        "Description": args.get('description'),
+        "Status": args.get('status'),
+        "Origin": args.get('origin'),
+        "Priority": args.get('priority'),
+        "Type": args.get('caseType')
+    }
+
+    results = client.createObject('Case', data)
+    if results.get('success', None):
+        case = client.getObject(f"{results.get('id')}")
+    else:
+        return_results(f"Unable to create case. Error Encountered was:{json.dumps(results.get('errors', None))}")
+
+    return objectToEntry(client, case)
+
+
+def update_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('oid', None) and args.get('caseNumber', None):
```

Comment:
I'm not sure whether the problem is that they were both given and you expect to get only one of them, or the problem is that they were both missing?  please write a better error message. 
In addition please raise an Exception instead of using the return_error() function.